src/components/Editor/AddProblemModal.tsx

```diff
@@ -31,98 +21,67 @@ async function addProblem(
       difficulty: 'N/A',
       isStarred: false,
       tags: ['Add Tags'],
-      solutionMetadata: parsed.solutionMetadata,
+      solutionMetadata: { kind: 'internal' },
     };
```

Comment:
this modal is also used for the "Add Problem" button in `module.problems.json`; in that use case, I think we do want the solutionMetadata to be autofilled to whatever the source-specific solution metadata is.

Reply:
sure! I'm not sure what to put for sources that aren't cf or usaco though (AC, CSES); any ideas?

Reply:
Maybe just "kind: none"? IDK what the current default is

Reply:
Also see https://github.com/cpinitiative/usaco-guide/issues/904  and https://github.com/cpinitiative/usaco-guide/blob/master/src/models/problem.ts#L323; there's some discussion about auto-generated solutions

Reply:
hm, none of these apply to the currently available parsers; will be putting kind: none for now!